apps/scan/backend/src/scanners/custom/app_scan.test.ts

```diff
@@ -536,3 +509,48 @@ test("scannerStatusToEvent's cases are exhaustive and can all be reached", () =>
   expect(eventCounts.get('SCANNER_READY_TO_EJECT')).toBeGreaterThan(0);
   expect(eventCounts.get('SCANNER_READY_TO_SCAN')).toBeGreaterThan(0);
 });
+
+test('scanning paused when election manager card is inserted', async () => {
```

Comment:
Any other guard conditions worth adding a test for? A few that feel relevant to real-world conditions:
- poll worker card inserted
- ballot bag needs replacement

Reply:
Added some more tests in e667a11.